build.groovy

```diff
@@ -270,6 +270,7 @@ options:
 	cli.v(longOpt:'verbose', 'Flag to turn on script trace')
 	cli.pv(longOpt:'preview', 'Supplemental flag indicating to run build in preview mode without processing the execute commands')
 	cli.cd(longOpt:'checkDatasets', 'Optional flag to validate the presense of the defined system datasets. ')
+	cli.cb(longOpt:'applicationCurrentBranch', args:1, 'Applications current Git branch. Used in pipeline builds to identify DBB Metadatastore objects.')
```

Comment:
Normally, the description would be something like:
> Application's current Git branch
Can you escape the single quote to make it look like this?